File: ProAgil.API/Models/Evento.cs
namespace ProAgil.API.Models
{
    public class Evento
    {
        public int EventoId { get; set; }

        public string local { get; set; }

        public string DataEvento { get; set; }

        public string Tema { get; set; }

        public int QtdePessoas { get; set; }

        public string Lote { get; set; }
    }
}

File: ProAgil-App/src/app/_models/Evento.ts
import { Lote } from "./Lote";
import { RedeSocial } from "./RedeSocial";
import { Palestrante } from "./Palestrante";


export interface Evento {
     id: number; 
     local: string; 
     dataEvento: Date;
     tema: string; 
     qtdePessoas: number; 
     imagemUrl: string; 
     telefone: string; 
     email: string; 
     lote: string; 
     lotes: Lote[];
     redesSociais: RedeSocial[];
     palestranteEventos: Palestrante[];
}
